ABOUT.md

```diff
@@ -0,0 +1,47 @@
+
+# About MineColonies 
+MineColonies is an interactive Town building mod that allows you to create your own thriving Town within Minecraft. It depicts real-life scenarios by providing you with different craftable options to build your own Town  and enhance your gaming experience. Featuring many NPC workers such as: Builders, Crafters, Farmers, Fishermen, Guards, Barracks, Miners, Smeltery, Baker, Cook/Restuarant, Deliveryman, Animal Herders and many more in development. As well as a specialized buildings such as: Warehouse, Citizen Huts and a Townhall. MineColonies gives you the ability to create a colony as rich and unique as every player.
+
+### Governance
+You have full control of how to build your Town. You determine how many workers you want or need, what type of workers, where they will live, hire or fire as you see fit. Creating a Town at your own pace. It is your Town to design and develop, you are the designer, administrator and boss always. 
+
+### Security and Replenishment
+Your Town deserves the best security you can have. Strategically add Guard Towers or Tower Barracks to protect your Town from the night mobs and barbarians that will attack in random nights. Give them the best weapons and armour your Town has to offer. They need to be fed well to work and protect it better. Keep a busy farmer to supply the baker and the cook at the restaurant, so no one goes hungry. 
+
+### Have fun, Play with Friends
+MineColonies is a very powerful multi-player system and an awesome tool for learning about Town designing, building and especially management. The permission system allows for collaborative team play in multi-player where you can assign different roles in a Town such as; Officer, Friend Neutral or even Hostile! Also have individual competing Colonies at the same time, all with a strong protection system for server management and Anti-Griefing. Fun for all! 
```

Comment:
Minecolonies (not MineColonies) and it has a very powerful multi-player, not is.
Learning about Town planning, building... would be better.

Also: colonies should be lower case, and it might better be, You can have individual, competing colonies.